flake8.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# There are 3 levels: F (FATAL), E (ERRORS) and W(WARN)
+# Everything is blacklisted here only due to excessive hits in existing code
+# base. The ignores should be removed in the severity level (e.g. F first, then E, then W)
+find -type f -name "*.py" -print | grep -v abichecker | grep -v openqa | \
```

Comment:
Ideally, this file should allow a git reference to be passed (which travis can provide) and use the git diff to only check the files that were changed. I suppose if the check runs fast and we end up cleaning everything this approach also works, but seems like it avoid you having to fix all the issue. If the pip8 has the ability to run on diffs then we can fix things whenever the line is changed, which would be the real preference.